src/pages/basic_data/app_admin/central_import/CentralImportPage.tsx

```diff
@@ -33,35 +39,29 @@ export const CentralImportPage = () => {
     history.push({ search: params.toString() });
   };
 
-  const publications = searchResults?.hits ?? [];
-
-  useEffect(() => {
-    if (searchResults?.hits.some(({ entityDescription }) => stringIncludesMathJax(entityDescription?.mainTitle))) {
-      typesetMathJax();
-    }
-  }, [searchResults]);
+  const searchResults = importCandidateQuery.data?.hits ?? [];
 
   return (
     <>
       <Typography variant="h3">{t('basic_data.central_import.publications')}</Typography>
-      {isLoadingSearchResults ? (
+      {importCandidateQuery.isLoading ? (
         <ListSkeleton minWidth={100} maxWidth={100} height={100} />
       ) : (
         searchResults && (
           <>
-            <Typography variant="subtitle1">{t('search.hits', { count: searchResults.size })}:</Typography>
+            <Typography variant="subtitle1">{t('search.hits', { count: searchResults.length })}:</Typography>
```

Comment:
`searchResults.length` vil vise lenge på returnerte hits, basert på paginering, så det vil typisk være opp mot 10.
For å vise count for alle treff:
```suggestion
            <Typography variant="subtitle1">{t('search.hits', { count: importCandidateQuery.data.size })}:</Typography>
```